Tutorials/whatiswot/website/docs/wot/td/introduction.md

```diff
@@ -1 +1,103 @@
+
+![consumer-definition](/img/13-Thing-Description/consumer.png)
+
+> Thing Descriptions are written using the JSON-LD format, making it both machine and human-readable. To understand this better, let's take a look at a simple example — a smart coffee machine — and break down the components in its Thing Description:
+
+```json
+{
+    "@context": ["https://www.w3.org/2022/wot/td/v1.1"
+               {"schema":"https://schema.org/"}], 
+    "id": "urn:uuid:0804d572-cce8-422a-bb7c-4412fcd56f06",
+    "title": "Smart Coffee Machine",
+    "description" : "Remote controllable coffee machine",
+    "schema:manufacturer":"ACME Corporation",
+    "securityDefinitions": { ... },
+    "security": { ... },
+    "properties": {
+        "coffeeBeansLeft": { ... }
+    },
+    "actions": {
+        "brewCoffee": { ... }
+    },
+    "events": {
+        "lowOnWater": { ... }
+    },
+    "links": [ ... ]
+}
+```
+## TD Components
+
+### Thing Metadata
+
+The Thing Description begins with the Thing's metadata. It provides the essential information about the device, like its unique ID, title, and description.
+
+```json
+    "id": "urn:uuid:0804d572-cce8-422a-bb7c-4412fcd56f06",
+    "title": "Smart Coffee Machine",
+    "description" : "Remote controllable coffee machine",
+```
+
+### Definition of Interaction Affordances
+
+In the definitions of Interaction Affordances, we specify ways a Consumer application can interact with the Thing, through its various properties, actions, and events exposed on its network interfaces.
+
+```json
+    "properties": {
+        "coffeeBeansLeft": { ... }
+    },
+    "actions": {
+        "brewCoffee": { ... }
+    },
+    "events": {
+        "lowOnWater": { ... }
+    }
+```
+
+> The example here shows the property `coffeeBeansLeft`, which describes the current state of the coffee beans; the action `brewCoffee`, which triggers brewing; and the event `lowOnWater`, which alerts the Consumer when the machine is low on water. We’ll explore Interaction Affordances in greater detail in the next tutorial of this series. 
+
+### Security Metadata
+
+The Security Metadata defines security mechanisms required to ensure authorized and secure interactions with the Thing by the Consumer application. This way the Consumer application knows what type of credentials are needed to execute different operations.
+
+```json
+    "securityDefinitions": { ... },
+    "security": { ... }
+```
+
+### Semantic Annotations
+
+To add keywords that are not part of the TD standard, Semantic Annotations can be added. These allow other systems to consistently interpret the device’s terms and functions.
+
+> Here, we annotate using `schema.org` to add the name of the manufacturer, which is not part of the core TD specification.
+
+```json
+"@context": ["https://www.w3.org/2022/wot/td/v1.1"
+               {"schema":"https://schema.org/"}]
+```
+
+```json
+    "schema:manufacturer":"ACME Corporation"
+```
```

Comment:
these should be merged but with `...` in between. Also the indentation should be fixed by expanding them